app/main.py

```diff
@@ -2,0 +1,25 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Команда повинна бути у форматі 'mv source destination'"
+        )
+    source = parts[1]
+    destination = parts[2]
+
+    if not os.path.exists(source):
+        raise FileNotFoundError(f"Файл {source} не існує")
+
+    if destination.endswith("/"):
+        os.makedirs(destination, exist_ok=True)
+        destination = os.path.join(destination, os.path.basename(source))
+    else:
+        destination_dir = os.path.dirname(destination)
+        if destination_dir and not os.path.exists(destination_dir):
+            os.makedirs(destination_dir, exist_ok=True)
+
+    shutil.move(source, destination)
```

Comment:
Code Efficiency: You are using 'shutil.move' to move the file, which is correct. But you should also handle the case when the 'source' file does not exist. The 'shutil.move' function will raise a 'FileNotFoundError' if the 'source' file does not exist.